src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs

```diff
@@ -204,8 +204,9 @@ public class SagaData : ContainSagaData
 
         protected override string CorrelationPropertyName
         {
-            //Explicitly not use expression body
+#pragma warning disable IDE0025 // Use expression body for properties
```

Comment:
Should we keep the comment as it was intentionally done this way?

Reply:
It's redundant with the new comment, so I removed it.

Reply:
I'm not sure. There's a difference between reading
> // Explicitly not use expression body

vs

> #pragma warning disable IDE0025 // Use expression body for properties

Reply:
We're having the same conversation of two different PRs.

The name of the class in the test already conveys that it's expecting a property with a body.

Reply:
I can add it back if you really think it's important.

Reply:
I think that'd be helpful. Thanks 🙂